### PR TITLE
Update data bubble navigation

### PR DESCRIPTION
**Pull request by bartaz**
## Done

Updated secondary navigation of MongoDB and OpenSearch in data bubble.

Copy doc: https://docs.google.com/document/d/1Y0dxbuar0UAPSCfILtP7fsEX-83yDh7Ha637FA9Wbik/edit?tab=t.0#heading=h.ib9c04garte2

Drive-by: some recent form updates started to cause secondary nav to dissapear on several pages. This was commented out here as a drive-by and reported separately in https://github.com/canonical/canonical.com/issues/1399

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Go to MongoDB page: https://canonical-com-1398.demos.haus/data/mongodb
  - make sure secondary navigation items are in order requested by the [copy doc](https://docs.google.com/document/d/1Y0dxbuar0UAPSCfILtP7fsEX-83yDh7Ha637FA9Wbik/edit?tab=t.0#heading=h.ib9c04garte2)
- Go to OpenSearch page: https://canonical-com-1398.demos.haus/data/opensearch
  - make sure secondary navigation items are in order requested by the [copy doc](https://docs.google.com/document/d/1Y0dxbuar0UAPSCfILtP7fsEX-83yDh7Ha637FA9Wbik/edit?tab=t.0#heading=h.ib9c04garte2)

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-15791

## Screenshots

image

### CODE REVIEW

File: webapp/app.py [hunk 1]
```diff
@@ -1207,4 +1207,7 @@ def set_form_rules():
             app.add_url_rule(path, view_func=render_form(form), endpoint=path)
 
 
-set_form_rules()
+# this causes secondary navigation to dissapear
```
Comment (bartaz) [inline webapp/app.py:1210]: A drive-by fix. Bug reported at: https://github.com/canonical/canonical.com/issues/1399

**Review by petesfrench — APPROVED** [review]
LGTM
